src/pages/docs/tutorials/tutorial-hello-enact/adding-sandstone-support/index.md

```diff
@@ -0,0 +1,81 @@
+
+## Higher-order Components
+
+In the simplest sense, Higher-order Components (HoCs) are functions that accept React components and return React components. They are useful for encapsulating common behavior, markup, and property transformations, which can later be applied to other components as needed.
+
+Enact provides several HoCs, as well as a factory method, `@enact/core/hoc`, to create your own. All Higher-order Components within Enact support an optional configuration object to
+customize their behavior. The first HoC we'll encounter is ThemeDecorator.
+
+> If you are coming from a background in frameworks that use mixins, Higher-order Components
+> may sound similar. While they are conceptually similar, HoCs provided better encapsulation,
+> resulting in fewer clashes between the implementations of disparate features.
+>
+> Dan Abramov has [offered a good perspective](https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750)
+> on the change to React to deprecate mixin support that may be informative.
+
+## ThemeDecorator
+
+`@enact/sandstone/ThemeDecorator` is a Higher-order Component that adds sandstone styling and several add-ons that are commonly used by sandstone applications: multi-resolution support, internationalization support, and Spotlight support.
+
+### Sandstone Styling
+
+The first change you'll notice after applying ThemeDecorator is the color and font changes which come from the `.sand` CSS class. Since Sandstone apps are designed to run full screen,
+ThemeDecorator also applies the global utility class `.enact-fit`, which uses absolute positioning to fill the available space.
+
+### Multi-Resolution Support
+
+Since Sandstone targets televisions that have different resolutions (720p, 1080p, 4k, and beyond!), Enact provides a set of tools to enable scaling applications for a consistent layout across resolutions. One part of the solution is [`less-plugin-resolution-independence`](https://github.com/enactjs/less-plugin-resolution-independence), a LESS plugin that adapts pixel measurements to relative measurements. The second part is `@enact/ui/resolution` which provides the capabilities to set the correct base size for the current resolution as well as utilities to work with resolution independent measurements in components.
+
+On top of this, Sandstone establishes a set of screen types for our supported television resolutions that are configured for the application by ThemeDecorator.
+
+### Internationalization Support
+
+In order to support delivery of applications to multiple locales, Enact integrates with the [iLib](https://sourceforge.net/projects/i18nlib/) internationalization library through our own module, `@enact/i18n`. This module provides the glue between the two code bases to correctly load locale resources and expose the results within an Enact app.
```

Comment:
The link for `iLib` is https://github.com/iLib-js/iLib.

Reply:
It will be fixed with your comment.